Focus/Lens.swift

```diff
@@ -38,11 +46,12 @@ public struct Lens<S, T, A, B> : LensType {
 	/// Gets the Indexed Costate Comonad Coalgebroid underlying the receiver.
 	private let _run : S -> IxStore<A, B, T>
 
+	/// Runs the lens on a structure to retrieve the underlying Indexed Costate Comonad Coalgebroid.
 	public func run(v : S) -> IxStore<A, B, T> {
 		return _run(v)
 	}
 
-	/// Produces a lens from an Indexed Costate Comonad Coalgebroid.
+	/// Produces a lens from a function yielding an Indexed Costate Comonad Coalgebroid.
```

Comment:
A coalgebra is *already* a function yielding a particular functor (in this case, the indexed store/costate comonad).  This "correction" is strictly *in*correct!